### ISSUE TITLE
joplin cli app crash

### ISSUE THREAD

**Issue by cjc7373** [archlinuxcn/repo#3085]
### 问题类型 / Type of issues

* 打包错误 / packaging error

### 受影响的软件包 / Affected packages

*  joplin

----
version: 2.9.17-2

log:
```bash
$ joplin
node:internal/modules/cjs/loader:1042
  throw err;
  ^

Error: Cannot find module '../../vendor/fountain.min.js'
Require stack:
- /usr/lib/node_modules/@joplin/renderer/MdToHtml/rules/fountain.js
- /usr/lib/node_modules/@joplin/renderer/MdToHtml.js
- /usr/lib/node_modules/@joplin/renderer/MarkupToHtml.js
- /usr/lib/node_modules/@joplin/renderer/index.js
- /usr/lib/node_modules/@joplin/lib/markdownUtils.js
- /usr/lib/node_modules/@joplin/lib/models/BaseItem.js
- /usr/lib/node_modules/@joplin/lib/file-api.js
- /usr/lib/node_modules/@joplin/lib/file-api-driver-local.js
- /usr/lib/node_modules/@joplin/lib/shim-init-node.js
- /usr/lib/node_modules/@joplin/lib/BaseApplication.js
- /usr/lib/node_modules/joplin/app.js
- /usr/lib/node_modules/joplin/main.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1039:15)
    at Module._load (node:internal/modules/cjs/loader:885:27)
    at Module.require (node:internal/modules/cjs/loader:1105:19)
    at require (node:internal/modules/cjs/helpers:103:18)
    at Object.<anonymous> (/usr/lib/node_modules/@joplin/renderer/MdToHtml/rules/fountain.js:3:18)
    at Module._compile (node:internal/modules/cjs/loader:1218:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1272:10)
    at Module.load (node:internal/modules/cjs/loader:1081:32)
    at Module._load (node:internal/modules/cjs/loader:922:12)
    at Module.require (node:internal/modules/cjs/loader:1105:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/usr/lib/node_modules/@joplin/renderer/MdToHtml/rules/fountain.js',
    '/usr/lib/node_modules/@joplin/renderer/MdToHtml.js',
    '/usr/lib/node_modules/@joplin/renderer/MarkupToHtml.js',
    '/usr/lib/node_modules/@joplin/renderer/index.js',
    '/usr/lib/node_modules/@joplin/lib/markdownUtils.js',
    '/usr/lib/node_modules/@joplin/lib/models/BaseItem.js',
    '/usr/lib/node_modules/@joplin/lib/file-api.js',
    '/usr/lib/node_modules/@joplin/lib/file-api-driver-local.js',
    '/usr/lib/node_modules/@joplin/lib/shim-init-node.js',
    '/usr/lib/node_modules/@joplin/lib/BaseApplication.js',
    '/usr/lib/node_modules/joplin/app.js',
    '/usr/lib/node_modules/joplin/main.js'
  ]
}

Node.js v19.3.0
```

**Comment by cuihaoleo** [comment]
Sorry for the crash. I only use GUI app so I didn't test the CLI one.

I haven't find the reason of the crash. The build system of Joplin is beyond my knowledge to debug. I need some more time to investigate. Before I solve the issue, please use the previous version: https://repo.archlinuxcn.org/x86_64/joplin-2.8.8-2-x86_64.pkg.tar.zst

**Comment by cuihaoleo** [comment]
This should have been fixed in joplin 2.9.17-4. Please reopen if not.

**Comment by cjc7373** [comment]
Thank you for your hard work! The app is working again.
